### PR TITLE
Feature/game view components integration

### PR DESCRIPTION
**Pull request by R1ZZU**

### CODE REVIEW

File: public/game/view-models/app-viewmodel.coffee [hunk 1]
```diff
@@ -5,6 +5,7 @@ Round = require '../models/round'
 MatchRendererViewModel = (require 'common/components/match-renderer').ViewModel
 UserPanelViewModel = (require 'common/components/user-panel').ViewModel
 TimespanViewModel = (require 'common/components/timespan').ViewModel
+OccurrenceIndicatorVM = (require 'common/components/occurrence-indicator').ViewModel
```
Comment (kirilknysh) [inline public/game/view-models/app-viewmodel.coffee]: Is `VM` correct? Shouldn't it be consistent with other `ViewModel`-s?

**Comment by kirilknysh** [comment]
I think we have to add `max-width` to each banned character to avoid:
image

File: public/common/components/occurrence-indicator/views/component.view.coffee [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require 'common/stylesheets/occurrence-indicator.styl'
```
Comment (DaQuirm) [inline public/common/components/occurrence-indicator/views/component.view.coffee]: Custom component styling should be on a higher level

File: public/game/view-models/app-viewmodel.coffee [hunk 3]
```diff
@@ -53,6 +54,15 @@ class AppViewModel
 		@roundTimerViewModel = new TimespanViewModel @countdown, TimespanViewModel.formats['m:ss']
 		@countdownViewModel = new TimespanViewModel @countdown, TimespanViewModel.formats['s']
 
+		@occurrenceIndicatorVMCell = new nx.Cell
+			'<-': [
+				@puzzle,
+				(puzzle) =>
+					vm = new OccurrenceIndicatorVM puzzle.banned_characters
```
Comment (DaQuirm) [inline public/game/view-models/app-viewmodel.coffee]: Why not make `patterns` a cell in the occurence indicator itself instead?

File: public/game/styles/game.styl [hunk 4]
```diff
@@ -15,22 +15,24 @@
...
-
-	.controls-selector
-		flex-grow 5
+		flex-wrap wrap
+		margin .8em 0
 
 	.timespan
 		font-size 3em
-		margin-left 0.5em
+		margin-left 0.3em
```
Comment (DaQuirm) [inline public/game/styles/game.styl]: We don't have any guidelines on leading zeroes, but let's keep it consistent :cat:

**Comment by DaQuirm** [comment]
Awesomeness extracted, therefore closing :smiley_cat:

**Pull request closed without merging**